src/app/features/public-form/public-form.component.html

```diff
@@ -1,7 +1,16 @@
 <mat-card>
   <mat-card-header>
-    <mat-card-title>{{ formConfig?.title }}</mat-card-title>
-    <mat-card-subtitle>{{ formConfig?.description }}</mat-card-subtitle>
+    <div>
+      <mat-card-title>{{ formConfig?.title }}</mat-card-title>
+      @if (formConfig?.logo) {
+        <app-display-img
+          [entity]="formConfig"
+          imgProperty="logo"
+          class="public-form-image"
```

Comment:
can we center this horizontally? It seems only left-aligned for me

Reply:
Do the below changes look good?
<img width="684" alt="image" src="https://github.com/user-attachments/assets/8ec671f1-241a-497c-8582-5dc0d65fe30f">

Reply:
great :+1: